compiler/src/iree/compiler/DispatchCreation/Passes.cpp

```diff
@@ -339,7 +343,9 @@ void buildDispatchCreationPassPipeline(
       /// `iree_tensor_ext.dispatch.workload.ordinal`
       ///   to map the captured operand to the position in the workload list.
       .addPass(
-          DispatchCreation::createMaterializeDefaultWorkgroupCountRegionPass);
+          DispatchCreation::createMaterializeDefaultWorkgroupCountRegionPass)
+      .addPass(createCSEPass)
```

Comment:
I typically think running canonicalization/cse works better, but I am fine with this.